modules/gapi/include/opencv2/gapi/gcommon.hpp

```diff
@@ -146,14 +165,16 @@ struct GAPI_EXPORTS_W_SIMPLE GCompileArg
     GCompileArg() = default;
 
     std::string tag;
+    std::function<void(gapi::s11n::IOStream&, const util::any&)> serialize;
+    util::any arg;
```

Comment:
Why it is public now? Not sure if there is a reason for this.

Reply:
For serialization routines to have opportunity to retrieve.
I can introduce getArg() method, should I?

Reply:
It is there already.

Reply:
...with casting though, let me double-check how you use it in the code.

Reply:
Fixed arg to be private, thanks!